benches/baseline.rs

```diff
@@ -0,0 +1,24 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use prometheus_client::metrics::counter::Counter;
+use prometheus_client::metrics::family::Family;
+
+pub fn baseline(c: &mut Criterion) {
+    c.bench_function("counter", |b| {
+        let counter: Counter = Counter::default();
+
+        b.iter(|| {
+            counter.inc();
+        })
+    });
```

Comment:
This should just boil down to however fast `AtomicU64::fetch_add` is. That said, won't hurt to have it here.

Reply:
Yep, seemed like a good baseline to me